src/Microsoft.OpenApi.OData.Reader/Operation/NavigationPropertyOperationHandler.cs

```diff
@@ -107,7 +123,7 @@ internal string GetOperationId(string prefix = null)
         /// <inheritdoc/>
         protected override void SetExternalDocs(OpenApiOperation operation)
         {
-            if (Context.Settings.ShowExternalDocs && Context.Model.GetLinkRecord(NavigationProperty, CustomLinkRel) is Link externalDocs)
```

Comment:
I might have mistaken this, but this looks liek it is replacing the old style with the new style, whereas we want the code to support both.

Reply:
This is noted. I'll update.